BigRoy/OpenPype
# TrayPublisher: colorspace overrides from defined ocio config

## Changelog Description
Adding enumerator attribute which is having aggregated options from active ocio config.

# Testing notes
## test global config
1. activate `project_settings/global/imageio/activate_global_color_management`
2. make sure some value is set here with correct path to ocio path `project_settings/global/imageio/ocio_config/filepath/0`
3. test and see values are aggregated form the config

## test host config override
1. activate `project_settings/global/imageio/activate_global_color_management`
2. activate `project_settings/traypublisher/imageio/activate_host_color_management` 
3. activate `project_settings/traypublisher/imageio/ocio_config/override_global_config`
4. make sure different config path is defined here ideally with different colorspace names `project_settings/traypublisher/imageio/ocio_config/filepath/0`
5. Reload Publisher
6. test and see values are aggregated form the config

## test deactivated colorspace management with default values
1. deactivate `project_settings/global/imageio/activate_global_color_management`
2. Reload Publisher
3. See default values in enumerator with 4 generic options.
